### ISSUE TITLE
toObject() tries to create an mongodb _id property for a domain class without this property

### ISSUE THREAD

**Issue by pigopl** [mpriatel/mongodb-grails#4]
Calling retrievedDoc.toObject() results in an attempted match of the _id mongodb property to a similarly named property in the domain object, which, of course, does not exist.  

If you create an _id property on the domain class it is happy.  

Is this intentional? (It is not documented like this)

**Comment by mpriatel** [comment]
Yes, you need to add an _id attribute to your domain objects. (and you're right, it's not documented =( ).

Future versions may automatically add the _id attribute.
